DESCRIPTION

```diff
@@ -34,7 +34,9 @@ Collate:
     minimize.R
     ConditionalPower.R
     ConditionalSampleSize.R
-    Smoothness_n2.R
+    SmoothnessN2.R
```

Comment:
yup, better naming! But why are you changing this in a pull request on Group-Sequential designs? Makes it harder to review... You can open as many PRs at the same time as you wish (please do not exploit that too much ;))